content/pt-br/immutable-infrastructure.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Infraestrutura Imutável
+status: Completed
+category: property
+tags: ["infraestrutura", "propriedade", "infraestrutura-como-codigo"]
+---
+
+Infraestrutura Imutável refere-se à infraestrutura de computadores
+([máquinas virtuais](/pt-br/virtual-machine/), [contêineres](/pt-br/container/), dispositivos de rede)
+que não pode ser alterada uma vez implantada.
```

Comment:
There's a word missing for it to make sense. 
What do you think of "alterada depois de implantada" or "alterada após a implantação" instead of "alterada uma vez implantada" ?